src/lib_c/x86_64-windows-msvc/c/win_nt.cr

```diff
@@ -0,0 +1,3 @@
+lib LibC
+  alias LPSTR = Char*
```

Comment:
Is it wise to alias LPSTR as a pointer to `Char`? 

In `WinNT.h` it is declared as a pointer of `CHAR` which is a 1-byte character only.
Reading a pointer of `Char` from windows API shouldn't be any issue but it will fail if it is populated from Crystal with multibyte characters.

Reply:
It should probably better be
```crystal
alias CHAR = UInt8
alias LPSTR = CHAR*
```

Reply:
Char is LibC::Char which is UInt8...

Reply:
Oh, I missed that. Still, would it make sense to add an uppercase alias `CHAR` as in `WinNT.h`?

Reply:
I'd rather not duplicate all the existing libc constants just in upper case just for windows.